<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>700</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>IMC - Login</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_enviar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(170, 0, 255);
	color: rgb(255, 255, 255);
	border-radius: 5px;
}
QPushButton:hover {
	color:  rgb(188, 188, 188);
	background-color: rgb(247, 10, 255);
}
QPushButton:pressed {
	color:  rgb(188, 188, 188);
	background-color: rgb(170, 0, 255);
}</string>
    </property>
    <property name="text">
     <string>LOG IN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_user">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
color: rgb(150, 150, 150);
padding: 5px;
border: 2px solid rgb(170, 0, 255);
border-radius: 4px;
}
QLineEdit:hover{
	border: 2px solid rgb(255, 0, 255)
}
QLineEdit:focus{
	color: rgb(0, 0, 0);
	border: 2px solid rgb(255, 0, 255)
}
</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="placeholderText">
     <string>USERNAME OR EMAIL</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_password">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
color: rgb(150, 150, 150);
padding: 5px;
border: 2px solid rgb(170, 0, 255);
border-radius: 4px;
}
QLineEdit:hover{
	border: 2px solid rgb(255, 0, 255)
}
QLineEdit:focus{
	color: rgb(0, 0, 0);
	border: 2px solid rgb(255, 0, 255)
}
</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>PASSWORD</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_login">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background: none;
	color: rgb(170, 0, 255);
}</string>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_erro_login_existe">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>451</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
border: 2px solid rgb(85, 0, 0);</string>
    </property>
    <property name="text">
     <string>This Login Already Exists!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_lbl_erro_login_existe">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(170, 0, 255);
	color: rgb(255, 255, 255);
	border-radius: 5px;
}
QPushButton:hover {
	color:  rgb(188, 188, 188);
	background-color: rgb(247, 10, 255);
}
QPushButton:pressed {
	color:  rgb(188, 188, 188);
	background-color: rgb(170, 0, 255);
}</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>ERROR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_erro_senha">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>451</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
border: 2px solid rgb(85, 0, 0);</string>
    </property>
    <property name="text">
     <string>Incorrect Password!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>ERROR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_lbl_erro_senha">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(170, 0, 255);
	color: rgb(255, 255, 255);
	border-radius: 5px;
}
QPushButton:hover {
	color:  rgb(188, 188, 188);
	background-color: rgb(247, 10, 255);
}
QPushButton:pressed {
	color:  rgb(188, 188, 188);
	background-color: rgb(170, 0, 255);
}</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_erro_login">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>451</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
border: 2px solid rgb(85, 0, 0);</string>
    </property>
    <property name="text">
     <string>Invalid Email/Username</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>219</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>ERROR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_lbl_erro_usuario">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(170, 0, 255);
	color: rgb(255, 255, 255);
	border-radius: 5px;
}
QPushButton:hover {
	color:  rgb(188, 188, 188);
	background-color: rgb(247, 10, 255);
}
QPushButton:pressed {
	color:  rgb(188, 188, 188);
	background-color: rgb(170, 0, 255);
}</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cadastrar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(170, 0, 255);
	color: rgb(255, 255, 255);
	border-radius: 5px;
}
QPushButton:hover {
	color:  rgb(188, 188, 188);
	background-color: rgb(247, 10, 255);
}
QPushButton:pressed {
	color:  rgb(188, 188, 188);
	background-color: rgb(170, 0, 255);
}</string>
    </property>
    <property name="text">
     <string>CREATE NEW ACCOUNT</string>
    </property>
   </widget>
   <zorder>lbl_erro_login_existe</zorder>
   <zorder>btn_enviar</zorder>
   <zorder>tb_user</zorder>
   <zorder>t_password</zorder>
   <zorder>lbl_login</zorder>
   <zorder>btn_lbl_erro_login_existe</zorder>
   <zorder>label</zorder>
   <zorder>btn_cadastrar</zorder>
   <zorder>lbl_erro_login</zorder>
   <zorder>label_3</zorder>
   <zorder>btn_lbl_erro_usuario</zorder>
   <zorder>lbl_erro_senha</zorder>
   <zorder>label_2</zorder>
   <zorder>btn_lbl_erro_senha</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tb_user</tabstop>
  <tabstop>t_password</tabstop>
  <tabstop>btn_enviar</tabstop>
  <tabstop>btn_lbl_erro_login_existe</tabstop>
  <tabstop>btn_lbl_erro_usuario</tabstop>
  <tabstop>btn_lbl_erro_senha</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
